### ISSUE TITLE
inflections dependency installed and then deleted immediately 

### ISSUE THREAD

**Issue by usharf** [syl20bnr/spacemacs#2956]
On `develop` I keep getting `inflection` installed and immediately removed...

```sh
Found 1 new package(s) to install...
--> fetching new package repository indexes...
--> installing dependency inflections... [1/1]
Found 1 orphan package(s) to delete...
--> deleting inflections... [1/1]

[227 packages loaded in 12.164s]
```

In messages:

```sh
(Spacemacs) --> installing dependency inflections... [1/1]
Making version-control local to inflections-autoloads.el while let-bound!
Generating autoloads for inflections.el...done
Saving file /Users/uri/.emacs.d/elpa/inflections-20121016.157/inflections-autoloads.el...
Wrote /Users/uri/.emacs.d/elpa/inflections-20121016.157/inflections-autoloads.el
(No changes need to be saved)
(No files need saving)
Checking /Users/uri/.emacs.d/elpa/inflections-20121016.157... [2 times]
Compiling /Users/uri/.emacs.d/elpa/inflections-20121016.157/inflections-pkg.el...done
Wrote /Users/uri/.emacs.d/elpa/inflections-20121016.157/inflections-pkg.elc
Checking /Users/uri/.emacs.d/elpa/inflections-20121016.157...
Compiling /Users/uri/.emacs.d/elpa/inflections-20121016.157/inflections.el...done
Wrote /Users/uri/.emacs.d/elpa/inflections-20121016.157/inflections.elc
Checking /Users/uri/.emacs.d/elpa/inflections-20121016.157...
...
(Spacemacs) --> deleting inflections... [1/1]
Package `inflections-20121016.157' deleted.
```

**Comment by justbur** [comment]
What layers are you using?

**Comment by bjeanes** [comment]
I'm getting this on master with both `inflections` and `hydra`:

```
Found 2 new package(s) to install...
--> fetching new package repository indexes...
--> installing hydra... [2/2]
Found 2 orphan package(s) to delete...
--> deleting inflections... [2/2]

[193 packages loaded in 16.466s]
```

I've tried re-installing spacemacs and clearing out my ELPA directory to no avail...

**Comment by syl20bnr** [comment]
Ok nailed it, I feel good now :-D
Should not happen anymore in develop, fix will be released in 0.104.0
